mock-certify-plugin/src/main/java/io.mosip.certify.mock.integration/service/MockVCIssuancePlugin.java

```diff
@@ -235,10 +250,47 @@ private static String getUTCDateTime() {
 	@Override
 	public VCResult<String> getVerifiableCredential(VCRequestDto vcRequestDto, String holderId,
 													Map<String, Object> identityDetails) throws VCIExchangeException {
+		String accessTokenHash = identityDetails.get(ACCESS_TOKEN_HASH).toString();
+        Map<String, Object> data = new HashMap<>();
+        String individualId;
+        try {
+			individualId = getIndividualId(getUserInfoTransaction(accessTokenHash));
+		} catch (Exception e) {
+			individualId = "123489";
```

Comment:
This was majorly for local setup since local spring (redis) cache is having encryption issues added a hardcoded value to avoid it, In remote cache is expected to work seamlessly. 

Will revert back this